Alignment/OfflineValidation/data/TkAlMap/Relative_TPolyLine_Pixel_BPIX_L1.txt

```diff
@@ -0,0 +1,162 @@
+# Xscale/Yscale
+# 362.795/150.41
+302055684 1.0 0.0 1.0 0.112672029785 0.949988285395 0.112672029785 0.949988285395 0.0
```

Comment:
@SenneVanPutte all of these files are huge and do not belong to CMSSW. 
I think we should rather put them in the cms-data repo.

@smuzaffar @mrodozov  can you:
1) create a repo in cms-data for `Alignment-OfflineValidation` ?
2) can you clarify if the format of the ASCII files introduced here are acceptable in the light of https://github.com/cms-sw/cmssw/issues/30158

Reply:
Ok sure let me know where exactly I can put them.

Reply:
The file format is fine as it is, every file is separate, they have extensions.
A single file should not be > 100mb (github treshold)
We have already ASCII files in data repos this is not an exception. Leave them as they are

Reply:
@mrodozov thanks for the confirmation. Can you confirm you can create a repo in `cms-data` for `Alignment-OfflineValidation`, so that Senne can move the files there?

Reply:
The repo has been created https://github.com/cms-data/Alignment-OfflineValidation. And yes please move the files there

Reply:
Added them here https://github.com/cms-data/Alignment-OfflineValidation/pull/1
Will now prepare the rest of the code for this setup